src/healthcareapis/azext_healthcareapis/generated/custom.py

```diff
@@ -43,14 +43,24 @@ def healthcareapis_service_create(client,
                                   export_configuration_storage_account_name=None,
                                   no_wait=False):
 
+    export_configuration = {
+        'storage_account_name': export_configuration_storage_account_name
+    }
+
+    service_identity_type = {
+        'principal_id': None,
+        'tenant_id': None,
+        'type': identity_type,
+    }
+
     properties = {
         'access_policies': access_policies,
         'cosmos_db_configuration': cosmos_db_configuration,
         'authentication_configuration': authentication_configuration,
         'cors_configuration': cors_configuration,
         'private_endpoint_connections': private_endpoint_connections,
         'public_network_access': public_network_access,
-        'export_configuration_storage_account_name': export_configuration_storage_account_name
+        'export_configuration': export_configuration
```

Comment:
all these code are in generated folder. If you want to manually override it, please add change in manual folder.